<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataconvert</class>
 <widget class="QMainWindow" name="dataconvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>岩石力学强度计算</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>模量转换参数：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>动静态杨氏模量转换</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Es</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="k_ymod">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="b_ymod">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>动静态泊松比转换</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>μs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="k_pois">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>μd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="b_pois">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_convert">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>660</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数据转换</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gp_conventional_log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>常规测井数据</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="con_logdata"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gp_handled_data">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>测井数据处理结果</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="handled_data"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>水平地应力计算公式选择：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>各向同性模型</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>212</width>
           <height>152</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>水平最大地应力(ζh)系数</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Biot系数（α）</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>地层孔隙压力系数（Pp）</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>水平最大地应力(ζH)系数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>测井段以上岩石密度（g/cm3)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>横向各向同性模型</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>212</width>
           <height>152</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>水平最大地应力(ζh)系数</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Biot系数（α）</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_12">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_13">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>地层孔隙压力系数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>水平最大地应力(ζH)系数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_14">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>非测井段岩石密度（g/cm3)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>地层参数设置：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>岩石骨架声波时差：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>岩石流体声波时差：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>岩石骨架密度：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="act_open_con"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_open_con"/>
  </widget>
  <action name="act_open_con">
   <property name="icon">
    <iconset resource="res_convert.qrc">
     <normaloff>:/toolbar/image/open.bmp</normaloff>:/toolbar/image/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开常规测井文件(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res_convert.qrc"/>
 </resources>
 <connections/>
</ui>
